http://sourceware.org/ml/gdb-cvs/2012-09/msg00145.html

### src/gdb/ChangeLog	2012/09/25 12:48:52	1.14697
### src/gdb/ChangeLog	2012/09/26 02:06:51	1.14698
## -1,3 +1,8 @@
+2012-09-26  Siddhesh Poyarekar  <siddhesh@redhat.com>
+
+	* breakpoint.c (invalidate_bp_value_on_memory_change): Expand
+	parameter LEN to ssize_t.
+
 2012-09-25  Siddhesh Poyarekar  <siddhesh@redhat.com>
 
 	* ada-valprint.c (ada_val_print_1): Eliminate single-use
--- src/gdb/breakpoint.c	2012/09/25 12:48:52	1.705
+++ src/gdb/breakpoint.c	2012/09/26 02:06:54	1.706
@@ -14718,7 +14718,7 @@
    GDB itself.  */
 
 static void
-invalidate_bp_value_on_memory_change (CORE_ADDR addr, int len,
+invalidate_bp_value_on_memory_change (CORE_ADDR addr, ssize_t len,
 				      const bfd_byte *data)
 {
   struct breakpoint *bp;
### src/gdb/doc/ChangeLog	2012/09/21 01:46:42	1.1370
### src/gdb/doc/ChangeLog	2012/09/26 02:06:55	1.1371
## -1,3 +1,7 @@
+2012-09-26  Siddhesh Poyarekar  <siddhesh@redhat.com>
+
+	* observer.texi (memory_changed): Expand parameter LEN to ssize_t.
+
 2012-09-21  Yao Qi  <yao@codesourcery.com>
             Pedro Alves  <palves@redhat.com>
 
--- src/gdb/doc/observer.texi	2012/09/21 01:46:43	1.40
+++ src/gdb/doc/observer.texi	2012/09/26 02:06:55	1.41
@@ -230,7 +230,7 @@
 This method is called immediately before freeing @var{inf}.
 @end deftypefun
 
-@deftypefun void memory_changed (CORE_ADDR @var{addr}, int @var{len}, const bfd_byte *@var{data})
+@deftypefun void memory_changed (CORE_ADDR @var{addr}, ssize_t @var{len}, const bfd_byte *@var{data})
 Bytes from @var{data} to @var{data} + @var{len} have been written
 to the current inferior at @var{addr}.
 @end deftypefun
